src/utility/Constants.js

```diff
@@ -8,3 +8,9 @@ export const NOTIFICATION_PERMISSIONS = Object.freeze({
   DENIED: 'denied',
 })
 export const DAYOF_COLLECTION = 'DayOf'
+export const SCREEN_BREAKPOINTS = {
```

Comment:
LOL i know you just moved them but maybe we should put these in the theme instead.

Reply:
wait why theme? wouldn't these be the same regardless of theme?

Reply:
mainly because theme stores constants related to styles and this is a constant relating to styling, also so we don't have to import the constant every time

Reply:
Down for either one!

Reply:
lets add it to the theme

Reply:
How should I pass this in for use for something like `maxWidthMediaQueries` in `Common.js`?

Reply:
ah its ok we can do this later lol